02-ServiceRegistration/src/PrismSample/ViewModels/MainPageViewModel.cs

```diff
@@ -1,11 +1,33 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
+﻿using PrismSample.Views;
+using Prism.Navigation;
+using Prism.Commands;
 using Prism.Mvvm;
 
 namespace PrismSample.ViewModels
 {
     public class MainPageViewModel : BindableBase
     {
+        private readonly INavigationService _navigationService;
+
+        public DelegateCommand GoToViewACommand { get; private set; }
+        public DelegateCommand GoToViewBCommand { get; private set; }
+
+        public MainPageViewModel(INavigationService navigationService)
+        {
+            _navigationService = navigationService;
+
+            GoToViewACommand = new DelegateCommand(GoToViewA);
+            GoToViewBCommand = new DelegateCommand(GoToViewB);
+        }
+
+        private async void GoToViewA()
+        {
+            await _navigationService.NavigateAsync(nameof(ViewAPage));
+        }
+
+        private async void GoToViewB()
+        {
+            await _navigationService.NavigateAsync(nameof(ViewBPage));
+        }
```

Comment:
use a magic string here... we want to keep this simple and `nameof(SomePage)` is a break of the MVVM pattern

Reply:
Done with the latest commit :)